expr/plan.go

```diff
@@ -3,28 +3,101 @@ package expr
 import (
 	"fmt"
 	"io"
+	"strings"
 
 	"github.com/grafana/metrictank/api/models"
 	"github.com/grafana/metrictank/consolidation"
 	"github.com/grafana/metrictank/errors"
 )
 
+type Optimizations struct {
+	PreNormalization bool
+	MDP              bool
+}
+
+func (o Optimizations) ApplyUserPrefs(s string) Optimizations {
+	if s == "" {
+		return o
+	}
+	o.PreNormalization = false
+	o.MDP = false
```

Comment:
Does this respect `http.mdp-optimization` from the config?

Reply:
yes but i came up with this patch last minute (to allow http params to override the settings) and mainly use it for testing. I will clarify it so that it makes more sense